src/crypto/OperationalKeystore.h

```diff
@@ -168,6 +168,26 @@ class OperationalKeystore
      */
     virtual CHIP_ERROR SignWithOpKeypair(FabricIndex fabricIndex, const ByteSpan & message,
                                          Crypto::P256ECDSASignature & outSignature) const = 0;
+
+    /**
+     * @brief Create an ephemeral keypair for use in session establishment.
+     *
+     * WARNING: There may be more than one of these in use concurrently, so implementations have to
+     *          be careful to avoid assuming a single instance.
+     *
+     * WARNING: The return value MUST be released by `ReleaseEphemeralKeypair`. This is because
+     *          Matter CHIPMem.h does not properly support UniquePtr in a way that would
+     *          safely allow classes derived from Crypto::P256Keypair to be released properly.
+     *
+     * @return a pointer to a dynamically P256Keypair (or derived class thereof), which may evaluate to nullptr
```

Comment:
```suggestion
     * @return a pointer to a P256Keypair (or derived class thereof), which may evaluate to nullptr
```
Nothing prevents the keystore from allocating these however it wants (e.g. a static fixed-size array).

Reply:
Fixed the wording.